packages/testing/src/api/directive.ts

```diff
@@ -32,6 +32,50 @@ interface DirectiveMocker {
   (directive: TransformerDirective, executionMock: TransformerMock): any
 }
 
+/**
+ *
+ * @description
+ * Used for writing directive tests for asynchronous directives e.g.
+ * - Transformer directives can be passed resolvedValue
+ * - Validator directives should check for errors thrown in certain situtations
+ *
+ * @example
+ *  const mockExecution = mockAsyncRedwoodDirective(myTransformer, {
+ *    context: currentUser,
+ *    resolvedValue: 'Original Value',
```

Comment:
The docs say `mockedResolvedValue` but here says `resolvedValue`—which should it be?